app/general-information/general-information.component.ts

```diff
@@ -56,15 +80,11 @@ export class GeneralInformationComponent implements OnInit {
 
     public ngOnInit(): void {
         this._sideDrawerTransition = new SlideInOnTopTransition();
-        this.imageSrc = "~/images/jacksonville.jpg";
-        this.title = "South West Gurupurnima 2018";
-        this.startDate = "Start Date: July 22 2018";
-        this.endDate = "End Date: July 28, 2018";
-        this.address = "Address: Jacksonville Florida, USA ";
+        // this.imageSrc = "~/images/jacksonville.jpg";
         // this.desc = "Our theme for Gurupurnima 2018 is global unity, and in this spirit, our tagline is YOUnity starts with YOU! As individuals, we can all play a role in making this event a success! " ;
         this.scheduleTitle = "Main Events";
         this.schedule = "Sunday July 22, 2018: Check-In\nMonday July 23, 2018: Gurupurnima starts\nFriday July 27, 2018: Gurupujan\nSaturday July 28, 2018: Gnanvidhi\nSunday July 29, 2018: Check-out after breakfast";
```

Comment:
Since we do not have a way to know which sessions should be "main sessions", even automating it would require hardcoding to get all important sessions
- For future i suggest we request dev team to add a isMainSession boolean so its easier for parsing
- Otherwise we we keep a list of main sessions on front-end and just pull those from API (which would be same as hardcoding as above)

Reply:
Sounds good. We can discuss some of these during GP with the team. I don't think they will have time to make these changes now.